include/abb_librws/rws_interface.h

```diff
@@ -49,6 +49,113 @@ namespace rws
 class RWSInterface
 {
 public:
+  /**
+   * \brief Type of a mechanical unit.
+   */
+  enum MechanicalUnitType
+  {
+    NONE,      ///< The unit has no type.
+    TCP_ROBOT, ///< The unit is a TCP robot.
+    ROBOT,     ///< The unit is a robot.
```

Comment:
So what's the difference?

Reply:
Clarified in 44e45c4

As far as I understand the ABB definitions, then a `TCP_ROBOT` can process commands in Cartesian space while a `ROBOT` cannot. Also, both a `TCP_ROBOT` and `ROBOT` always has more than one joint.

An [IRBP A](https://new.abb.com/products/robotics/application-equipment-and-accessories/workpiece-positioners/irbp-a) positioner, consisting of two external joints, is for example considered as a `ROBOT`.